web_services/profile/app.js

```diff
@@ -50,6 +97,34 @@ app.post("/upload", function (req, res, next) {
   }
 });
 
+app.get("/download/:channel/:uuid", function (req, res, next) {
```

Comment:
Is this easier for the client than downloading all the profiles at once?

Reply:
Bump @TripleSpeeder

Reply:
@adamstallard I think it can be a nice optimization to add a batch download operation. I will add an issue on the mobile app side to add that, but no need to change for this PR.